boards/mavrinux.cpp

```diff
@@ -76,7 +76,8 @@ Mavrinux::Mavrinux(mavrinux_conf_t config):
     led(),
     mavlink_serial(config.serial_udp_config),
     file_flash(),
-    config_(config)
+    config_(config),
+    state_display_mavrinux_()
```

Comment:
Additionally, this gives a **warning** !!
because of the order of the members and constructor not matching.
Try to be really systematic => no errors neither warning tolerated ! :)